src/ controllers/trainee/Controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Request, Response } from 'express';
+
+class TraineeController {
+    static instance;
+    static getInstance = () => {
+        if (TraineeController.instance) {
+            return TraineeController.instance;
+        }
+        TraineeController.instance = new TraineeController();
+        return TraineeController.instance;
+    }
```

Comment:
No need of this code snippet, just export the new instance of the trainee controller while exporting

Reply:
Done